schemas/com.growthbook/experiment_viewed/jsonschema/1-0-0

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for an GrowthBook experiment viewed event",
+  "self": {
+    "vendor": "com.growthbook",
```

Comment:
`growthbook.com` doesn't resolve to anything, should this be under `io.growthbook`?

Reply:
Fixed